Add JetBrains themes

artemevsevev opened the pull request:
Light and dark themes: https://github.com/artemevsevev/zed-theme-jetbrains

notpeter commented:
Curious how this differs from the existing jetbrains style theme:
- https://github.com/kpitt/zed-theme-intellij-newui

artemevsevev commented:
I used colors from the current version of JetBrains IDE. In https://github.com/kpitt/zed-theme-intellij-newui there are differences in colors of syntax and some interface elements.

Added colors for several new syntax elements (lifetime, macros)

Added a light theme.
